File: api/DatabaseAPI/Models/Activity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace DatabaseAPI.Models
{
    public class Activity
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public List<Group> Listeners { get; set; }
        public string FIO { get; set; }
        public string Link { get; set; }

        [JsonIgnore]
        public DaySchedule Schedule { get; set; }

        [JsonIgnore] 
        public int ScheduleId { get; set; }
    }
}
  

File: mobile/UniversityScheduler/app/src/main/java/com/jerael/universityscheduler/models/Activity.kt
package com.jerael.universityscheduler.models

data class Activity(

    var id: Int? = null,

    var name: String? = null,

    var startTime: String? = null,

    var endTime: String? = null,

    var listeners: List<Listener>? = null,

    var fio: String? = null,

    var link: String? = null

)
